README.md

```diff
@@ -17,8 +17,21 @@ You can add Reactist to your project by installing it from npm:
 npm install @doist/reactist
 ```
 
+To import a component within your code:
+
+```js
+import loading from "@doist/reactist/lib/loading"
+```
+
+You also need to load the CSS file of the component somewhere in your app. For instance:
+
+```
+<link rel="stylesheet" type="text/css" href="./node_modules/@doist/reactist/lib/Loading.css">
```

Comment:
Maybe we also mention that there's one `reactist.css` that includes css for all components?